core/src/apps/common/passphrase.py

```diff
@@ -26,6 +26,7 @@ async def get(ctx: wire.Context) -> str:
 
 
 async def _request_from_user(ctx: wire.Context) -> str:
+    workflow.close_others()  # request exclusive UI access
```

Comment:
It's because of `_entry_dialog()`. We are now in the passphrase flow and that always puts _something_ on screen

Reply:
I mean why it is not in `_entry_dialog` then. In `_entry_dialog` for `_request_on_host` and `_request_on_device` has it in `button_request`.

Reply:
It makes more sense to me like this.

Before (no "request UI exclusivity" exists): we draw things as needed, and as a special case, in `_entry_dialog` we _also_ need to make sure that default is killed.

Now: in `_request_from_user` we are entering the UI-enabled part of passphrase entry. Request UI exclusivity because we need it in either case: the fact that `button_request()` does the same thing later becomes an implementation detail.